### PR TITLE
update api docs for bclient

### PR DESCRIPTION
**Pull request by bucko13**
Updated all `bcoin.http.client` to NodeClient and WalletClient from `bclient.

### CODE REVIEW

File: api-docs-slate/source/includes/_wallet.md [hunk 1]
```diff
@@ -1,4 +1,38 @@
...
+`npm i -g bclient` gives you access to the cli. You can also install locally
+to your project.
+
+Note that when using it in your own program, you will need to explicitly
+pass in a port option. The easiest way to do this is with `bcoin.Network`.
+
+You can create a client for a specific wallet (and be compatible with the old
+api) with the `wallet` method on `WalletClient` class.
+
+```shell--curl
```
Comment (tynes) [inline api-docs-slate/source/includes/_wallet.md]: Extra formatting block here

File: api-docs-slate/source/includes/_wallet.md [hunk 2]
```diff
@@ -1,4 +1,38 @@
...
+const { WalletClient } = require('bclient');
+const { Network } = require('bcoin');
+const network = Network.get('testnet');
+
+const walletClient = new WalletClient({
+  port: network.walletPort,
+  network: network.type
+});
+
+const id = 'primary'; // or 'foo' or whatever
```
Comment (tynes) [inline api-docs-slate/source/includes/_wallet.md]: Maybe change comment to something less ambiguous?  Something like
`// name of wallet` or `// any wallet name` ?

File: api-docs-slate/source/includes/_coin.md [hunk 3]
```diff
@@ -32,7 +32,8 @@ bcoin-cli coin $hash $index
 ```
 
 ```javascript
-const client = new bcoin.http.Client({
+const { NodeClient } = require('bclient');
```
[the remark this answers is not in the document]
Comment (bucko13) [reply api-docs-slate/source/includes/_coin.md]: Fixed! In bPanel we actually do it this way, based off of the baseline prettier configs, but I think for the official bcoin docs this makes sense. The justification for the difference I think is that one is meant as a browser env application and this/bcoin's formatting is for more node environments.

**Comment by bucko13** [comment]
Going to merge this in now! Let me know if there are any other requests and I can update in separate PR.

**Pull request merged**